src/gbd_mapping/risk_factor_template.py

```diff
@@ -27,11 +28,14 @@ def __init__(self,
                  distribution: str,
                  paf_calculation_type: str,
                  restrictions: Restrictions,
-                 missing_exposure: Union['bool', 'None'],
-                 missing_rr: Union['bool', 'None'],
-                 rr_less_than_1: Union['bool', 'None'],
-                 missing_paf: Union['bool', 'None'],
-                 paf_outside_0_1: Union['bool', 'None'],
+                 exposure_exists: Union["bool", None],
```

Comment:
So I don't really care one way or the other, but I'd like to be consistent between all entities: do we want to do all the exists flags in a block and then all the in range flags in a block, etc. or do we want to group by measure so all the flags for a single measure are in one block?

Reply:
I tend to logically group measures, so I did it this way. No preference on my end